packages/core/src/icon/icon.stories.ts

```diff
@@ -393,10 +393,10 @@ export const legacy = () => {
     <h2>Direction</h2>
     <!-- @ts-ignore -->
     <div>
-      <clr-icon shape="caret" dir="up"></clr-icon>
```

Comment:
This story is here to verify that `dir` still works. It's a test of the legacy API. So using `dir` here is actually preferred. Other than that, this PR lgtm.

Reply:
🤦 My bad, let me revert that. Sorry!

Reply:
Done!